tests/fixtures/sleeps-composefile/one-exits.yml

```diff
@@ -0,0 +1,6 @@
+simple:
+  image: busybox:latest
+  command: sleep 200
+another:
+  image: busybox:latest
+  command: sleep 0.5
```

Comment:
Scratch that, I have a better idea. How about both services use `top` as the command. We use `top` because it exits properly on `SIGTERM`. I'll cover the testing changes above.